FirebaseMLModelDownloader/Sources/ModelInfoRetriever.swift

```diff
@@ -199,6 +194,15 @@ extension ModelInfoRetriever {
     }
   }
 
+  /// Date formatter to parse download URL expiry time.
+  private static func getDateFormatter() -> DateFormatter {
+    let dateFormatter = DateFormatter()
+    dateFormatter.locale = Locale(identifier: "en-US_POSIX")
+    dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"
```

Comment:
nit: Looks like it can be parsed with [ISO8601DateFormatter](https://developer.apple.com/documentation/foundation/iso8601dateformatter). If so, it would be preferable to use ISO8601DateFormatter when available and fallback to manual date format. In this case the fallback will be clearly visible to be removed once we bump supported OS versions.

Reply:
Done, I thought this didn't support milliSeconds but it does with iOS 11+.